sdk/src/AbstractMain.cpp

```diff
@@ -225,19 +218,19 @@ int AbstractMain::run(int in_argc, char** in_argv)
    if (ret != 0)
       return ret;
 
-   // Remove the stderr log destination.
-   removeLogDestination(stderrLogDest->getId());
-
-   if (options.getLogLevel() > LogLevel::INFO)
-   {
-      addLogDestination(
+   addLogDestination(
          std::unique_ptr<ILogDestination>(
             new FileLogDestination(
-               3,
+               "MainLogDestination",
                options.getLogLevel(),
+               LogMessageFormatType::PRETTY,
                getProgramId(),
-               options.getScratchPath())));
-   }
+               logging::FileLogOptions(options.getLoggingDir())
+               )));
+
+   logging::logInfoMessage("Starting plugin...");
```

Comment:
Let's move this back closer to where it was originally, so right after the call to `addLogDestination(stderrLogDestination)`.